js/widgets/temperament.js

```diff
@@ -1760,8 +1763,7 @@ class TemperamentWidget {
             }
         }
 
-        // Global value
-        OCTAVERATIO = this.powerBase;
+        const OctaveRatio = this.powerBase;
```

Comment:
global constant ? simple rename?

Reply:
Yeah, it worked here perfectly.
Actually, making the variable `OCTAVERATIO` as `let` in the musicutils.js file didn't work. Thus, this step.